apps/workspaces/apis/clone_settings/serializers.py

```diff
@@ -0,0 +1,81 @@
+from apps.workspaces.models import Workspace
+from rest_framework import serializers
+
+from django.db import transaction
+
+from apps.workspaces.apis.export_settings.serializers import ExportSettingsSerializer, \
+    ReadWriteSerializerMethodField
+
+from apps.workspaces.apis.import_settings.serializers import ImportSettingsSerializer
+from apps.workspaces.apis.advanced_settings.serializers import AdvancedSettingsSerializer
+
+
+class CloneSettingsSerializer(serializers.ModelSerializer):
+    export_settings = ReadWriteSerializerMethodField()
```

Comment:
Check if we can do something like this and that way we'll not need to explicitly add validations, not sure if we can do this, but it will be much cleaner that way
```suggestion
    export_settings = ReadWriteSerializerMethodField(null=False, blank=False)
```

Reply:
this doesn't seem to be possible, have checked it out